src/schema/rules/sidecars/pet.yaml

```diff
@@ -79,19 +79,19 @@ PETRadioChemistry:
       level: recommended
     InfusionRadioactivity:
       level: recommended
-      level_addendum: required if ModeOfAdministration is `'bolus-infusion'`
+      description_addendum: required if ModeOfAdministration is `'bolus-infusion'`
```

Comment:
I think this should be level_addenum, since it describes how the requirement level changes based on the context.

Reply:
Oh, I thought we scrapped the level_addendum